### DISCUSSION TITLE
How to use Tab as div onChange using typescript?

### DISCUSSION THREAD

**Discussion by devnomic** [tailwindlabs/headlessui#1261] [Help] [answered]
I have TabGroup as div
````
<Tab.Group
        as="div"
        className="flex flex-col w-full"
        defaultIndex={0}
        selectedIndex={selectedIndex}
        onChange={(index: number): void => {}}
      >
````
Typescript wants me to use `(React.FormEventHandler<HTMLDivElement> & ((index: number) => void))`

It's a bitwise & on `function`, how do i satisfy the typescript ?
How to declare this function?
Thanks

<img width="985" alt="Screen Shot 2022-03-20 at 15 55 35" src="https://user-images.githubusercontent.com/101681592/159155581-06505d03-7ebb-4776-b903-37c9944dc730.png">

**Comment by devnomic** [comment, marked as answer]
This seems satisfy the ts
````
onChange={(index: number | React.FormEvent<HTMLDivElement>) => {
}}
````

**Comment by mohammadzolfali5** [comment]
[375242](mohammadzolfali54@gmail.com)
